src/Uno/Prism.Uno/Prism.Uno.csproj

```diff
@@ -0,0 +1,77 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="MSBuild.Sdk.Extras">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;xamarinios10;monoandroid90;xamarinmac20;uap10.0.16299</TargetFrameworks>
+    <GenerateLibraryLayout>true</GenerateLibraryLayout>
+    <RootNamespace>Prism</RootNamespace>
+    <DefineConstants>$(DefineConstants);HAS_WINUI</DefineConstants>
+    <NoWarn>$(NoWarn);1591</NoWarn>
+    <PackageId>Prism.Uno</PackageId>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
+    <DefineConstants>$(DefineConstants);__WASM__</DefineConstants>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(TargetFramework)'=='xamarinios10' or '$(TargetFramework)'=='xamarinmac20' or '$(TargetFramework)'=='monoandroid90' or '$(TargetFramework)'=='netstandard2.0'">
+    <PackageReference Include="Uno.UI" Version="2.0.528" />
+    <PackageReference Include="Uno.Core" Version="2.0.0-dev.5" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)'=='uap10.0.16299'">
+    <PackageReference Include="Microsoft.Xaml.Behaviors.Uwp.Managed" Version="2.0.1" />
+    <PackageReference Include="CommonServiceLocator" Version="2.0.5" />
+  </ItemGroup>
+  <ItemGroup>
+    <Page Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
+    <Compile Update="**\*.xaml.cs">
+      <DependentUpon>%(Filename)</DependentUpon>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <UpToDateCheckInput Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\..\Wpf\Prism.Wpf\**\*.cs" Exclude="..\..\Wpf\Prism.Wpf\bin\**\*.cs;..\..\Wpf\Prism.Wpf\obj\**\*.cs" />
+    <Compile Remove="**\*.net45.cs" />
+    <Compile Remove="**\*.netcore.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="..\..\Wpf\Prism.Wpf\Interactivity\InvokeCommandAction.cs" />
+    <Compile Remove="..\..\Wpf\Prism.Wpf\Modularity\ConfigurationModuleCatalog.Desktop.cs" />
+    <Compile Remove="..\..\Wpf\Prism.Wpf\Modularity\ConfigurationStore.Desktop.cs" />
+    <Compile Remove="..\..\Wpf\Prism.Wpf\Modularity\IConfigurationStore.Desktop.cs" />
+    <Compile Remove="..\..\Wpf\Prism.Wpf\Modularity\ModuleConfigurationElement.Desktop.cs" />
+    <Compile Remove="..\..\Wpf\Prism.Wpf\Modularity\ModuleConfigurationElementCollection.Desktop.cs" />
+    <Compile Remove="..\..\Wpf\Prism.Wpf\Modularity\ModuleDependencyCollection.Desktop.cs" />
+    <Compile Remove="..\..\Wpf\Prism.Wpf\Modularity\ModuleDependencyConfigurationElement.Desktop.cs" />
+    <Compile Remove="..\..\Wpf\Prism.Wpf\Modularity\ModulesConfigurationSection.Desktop.cs" />
```

Comment:
What's the difference there? Dynamic loading is supported for Wasm at this point, same for macOS, and any .NET Core based platform.

Reply:
It isn't supported crossplat at this point... as I understand it from our conversation yesterday it wouldn't be supported on UWP either. True there are exceptions but I think it would be best to align this with the Forms implementation which is already taking these various platforms into account.

Reply:
The implementation is now the one from Forms.